packages/lit-xml/.npmignore

```diff
@@ -1,4 +1,5 @@
 **
 !dist/src/**/*
-*.map
+!src/**/*.ts
+tsconfig.json
```

Comment:
why us this line needed? It should already be excluded with the top level `**`

Reply:
Yeah I'm not sure to be honest

Reply:
Updated it